test/e2e/network/netpol/network_policy.go

```diff
@@ -132,7 +132,7 @@ var _ = common.SIGDescribe("Netpol", func() {
 			}
 		})
 
-		ginkgo.It("should support a 'default-deny-ingress' policy [Feature:NetworkPolicy]", func() {
+		ginkgo.It("SMOKE should support a 'default-deny-ingress' policy [Feature:NetworkPolicy]", func() {
```

Comment:
Revert this?

Reply:
sure lets just keep it while were testing bc its easy to just do `ginkgo.focus=SMOKE` and run a test that things arent broken :)

Reply:
i removed this now that i think were mostly ready to mertge here